evm-types.md

```diff
@@ -581,26 +581,26 @@ The local memory of execution is a byte-array (instead of a word-array).
 
     syntax ByteArray ::= #asByteStack ( Int ) [function]
  // ----------------------------------------------------
-    rule #asByteStack(W) => Int2Bytes(W, BE, Unsigned)
+    rule #asByteStack(W) => Int2Bytes(W, BE, Unsigned) [concrete]
 
-    syntax ByteArray ::= ByteArray "++" ByteArray [function, right, klabel(_++_WS), smtlib(_plusWS_)]
+    syntax ByteArray ::= ByteArray "++" ByteArray [function, functional, right, klabel(_++_WS), smtlib(_plusWS_)]
  // -------------------------------------------------------------------------------------------------
-    rule WS ++ WS' => WS +Bytes WS'
+    rule WS ++ WS' => WS +Bytes WS' [concrete]
 
     syntax ByteArray ::= ByteArray "[" Int ".." Int "]" [function]
  // --------------------------------------------------------------
-    rule WS [ START .. WIDTH ] => substrBytes(padRightBytes(WS, START +Int WIDTH, 0), START, START +Int WIDTH) requires START <Int #sizeByteArray(WS)
-    rule WS [ START .. WIDTH ] => padRightBytes(.Bytes, WIDTH, 0)                                              [owise]
+    rule WS [ START .. WIDTH ] => substrBytes(padRightBytes(WS, START +Int WIDTH, 0), START, START +Int WIDTH) requires START <Int #sizeByteArray(WS) [concrete]
+    rule WS [ START .. WIDTH ] => padRightBytes(.Bytes, WIDTH, 0)                                                                                     [owise, concrete]
 
-    syntax Int ::= #sizeByteArray ( ByteArray ) [function, functional]
+    syntax Int ::= #sizeByteArray ( ByteArray ) [function, functional, klabel(sizeWordStackAux), smtlib(sizeByteArray)]
```

Comment:
why call it `sizeWordStackAux`? for the java backend?

Reply:
I think this was a copy-paste error.